js/ui/overlay/ui.overlay.js

```diff
@@ -240,6 +243,14 @@ const Overlay = Widget.inherit({
         });
     },
 
+    ctor: function(element, options) {
+        this.callBase(element, options);
+
+        if(options && options.copyRootClassesToWrapper && !options.hideCopyClassesDeprecationWarning) {
+            errors.log('W0001', this.NAME, 'copyRootClassesToWrapper', '21.2', 'Use the "wrapperAttr" option instead');
```

Comment:
What about runtime change?

Reply:
earlier we desided not to proccess runtime `copyRootClassesToWrapper`